modules/ethereum/src/import.rs

```diff
@@ -31,13 +31,15 @@ use sp_std::{collections::btree_map::BTreeMap, prelude::*};
 /// we have NOT imported.
 /// Returns error if fatal error has occured during import. Some valid headers may be
 /// imported in this case.
-pub fn import_headers<S: Storage, PS: PruningStrategy>(
+#[allow(clippy::too_many_arguments)]
```

Comment:
I actually totally agree with clippy here. Some sort of 'import_ctx' struct should be introduced. This would kind of restructure the whole solution, though. So maybe this could/should be done as a separate issue.

Reply:
Done